tensorboard/components/tf_dashboard_common/tf-dashboard-layout.html

```diff
@@ -33,14 +33,18 @@
     </div>
     <style include="scrollbar-style"></style>
     <style>
+      :host {
+        display: flex;
+        flex-direction: row;
+        height: 100%;
+      }
+
       #sidebar {
-        width: inherit;
+        flex: 0 0 25%;
```

Comment:
Just to check my understanding, this CSS is what will now define the sidebar width and it's defined to be 25% of the parent (dashboard) width, bounded to a min of 270px and a max of 350px?  Is that correct?

Reply:
Yup! That's correct!